ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -119,7 +119,7 @@ conformance_test(
     server_args = [
         "--contract-id-seeding=testing-weak",
         "--participant participant-id=example,port=6865",
-        "--batching enable=true,max-queue-size=500",
+        "--batching enable=true,max-queue-size=500,max-batch-size-bytes=262144,max-concurrent-commits=2",
```

Comment:
If we need these parameters in order for this to work, perhaps they shouldn't have defaults. The ledger could specify its own defaults if you like, but having them in kvutils itself is quite dangerous.

Reply:
We provide these defaults to get started easily. However, I agree that it would make sense to specify good defaults per ledger. Via `LedgerFactory` you may simply override the defaults used for batching, i.e., these are not enforced by kvutils in any way.
I'll add some more documentation to these settings in a separate PR.